lib/enigma.rb

```diff
@@ -2,28 +2,32 @@
 require 'pry'
 
 class Enigma
-  attr_reader :message, :key, :date
+  attr_reader :message,
+              :key,
+              :date,
+              :encryption,
+              :decryption
   def initialize
     @message = message
     @key = key
     @date = date
+    @encryption = Hash.new
```

Comment:
Initialized encryption and decryption so they can be called in the runner files.